spec/unit/views/components/panel_spec.rb

```diff
@@ -17,17 +17,17 @@
   end
 
   it "should add panel actions to the panel header" do
-    link = panel_html.find('h3 > div.header_action a')
+    link = panel_html.find('div.panel-heading-actions a')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.